app/scripts/services/SongPlayer.js

```diff
@@ -60,6 +60,13 @@
         * @type {Object}
         */ 
         SongPlayer.currentSong = null;
+        
+        /**
+        * @desc volume control
+        * @type {Number}
+        */ 
+        SongPlayer.volume = 80;
```

Comment:
Ahh, i see what happened. So the volume was changing but SongPlayer.volume was always 80. I fixed this now